transifex/utils.py

```diff
@@ -117,7 +117,7 @@ class Repo:
     # Make 9 attempts to merge the PR (the original attempt plus 8 retries), sleeping for 5 minutes between each
```

Comment:
Update comment.  It now makes 13 attempts

Reply:
Any reason why we needed to bump this?

Reply:
@bderusha my guess is that edx-platform takes a lot longer to run jenkins/CI checks than most of our other repos.

Reply:
yes i tried it with existing and it was not able to merge because checks took too mucch time.